app/utils.py

```diff
@@ -41,8 +42,31 @@ def get_location_info_from_coordinates(latitude, longitude, language='es'):
         logger.error(f"Error when doing reverse geo-coding {e}")
         return False, address, postal_code, city, region, country
 
+def get_location_info_from_coordinates(latitude, longitude, language='es'):
+    address, postal_code, city, region, country = '', '', '', '', ''
+    logger.info(f"Going to look for information about the location with latitude {latitude} and "
+                f"longitude {longitude}")          
+    try:
+        content = urllib.request.urlopen("https://nominatim.openstreetmap.org/reverse?format=jsonv2&lat=" + str(latitude) + "&lon="+str(longitude)).read() #consulto esta api de OSM 
+        jsonResult=json.loads(content.decode(), parse_float=float)
+        address = jsonResult['address']['road']
+        postal_code = jsonResult['address']['postcode']
+        city = jsonResult['address']['city']
+        try:
+            region = jsonResult['address']['state']
```

Comment:
Cambiar. Para comprobar si una clave existe en un dictionary la forma "pythonica" de hacerlo es
`if CLAVE in DICTIONARY`.
También comprobar que las claves road, postcode, y city existen. Si por algún motivo el API de OSM cambia y ya no se envian estas claves el código va a fallar.